modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java

```diff
@@ -392,18 +392,12 @@ public Reader getTemplate(String name) {
                             }
                             File outputFile = new File(outputFilename);
                             OutputStream out = new FileOutputStream(outputFile, false);
-                            if (in != null && out != null) {
+                            if (in != null) {
```

Comment:
The "out" object can't be null at this location since the constructor of the class ```FileOutputStream``` can't give a null object (but may throw exception if error occur).

```java
    public FileOutputStream(File file, boolean append)
        throws FileNotFoundException
    {
        String name = (file != null ? file.getPath() : null);
        SecurityManager security = System.getSecurityManager();
        if (security != null) {
            security.checkWrite(name);
        }
        if (name == null) {
            throw new NullPointerException();
        }
        if (file.isInvalid()) {
            throw new FileNotFoundException("Invalid file path");
        }
        this.fd = new FileDescriptor();
        this.append = append;
        this.path = name;
        fd.incrementAndGetUseCount();
        open(name, append);
    }
```

Can you please take a look at https://github.com/swagger-api/swagger-codegen/issues/1888 before this PR ? I'm currently blocked in my work on Codgen CXF Support :(